src/components/bcros/filing/CommonTemplate.vue

```diff
@@ -77,6 +77,14 @@
         </template>
       </slot>
 
+      <slot name="document-record">
+        <!-- if we have document records(aka filings staff completed on behalf of a client), show them -->
+        <template v-if="!isStaffFiling(filing) && enableDocumentRecords && !!getDocumentId">
```

Comment:
Do you need to check the FF here, or would getDocumentId be falsy without the FF anyway?

Reply:
I think the latter is true yes, but the computed value also considers that the current user is staff.

Reply:
Which computed value considers whether the current user is staff?

I'm simply thinking you don't need to check the FF here.

I just really paid attention to "isStaffFiling" and I'm thinking it doesn't do what you want. If I'm not mistaken it returns True for Court Order, Registrar's Notation, etc. What you really want here isn't the type of filing, but who filed it, right?

Maybe you don't need to check "isStaffFiling" either, here, because the "simple staff filings" won't have a DRS section, while fuller filings will (when completed by staff).

![image](https://github.com/user-attachments/assets/f382b9f4-cd0d-489f-8515-0ba4916b1af6)

I'm OK to take this offline if you want to chat directly about it.

Reply:
We will want to check if the current account is Staff. The DRS UI isn't directly exposed to the public.

Reply:
But yes i agree, Staff Filings that have no consumerReferenceId will not appear either way, so we can do without that condition i believe.

Reply:
> We will want to check if the current account is Staff. The DRS UI isn't directly exposed to the public.

I agree, but that's not what "isStaffFiling" does.

> But yes i agree, Staff Filings that have no consumerReferenceId will not appear either way, so we can do without that condition i believe.

Cool. And if you didn't load documents for non-staff, you wouldn't need to check that either ;)

Reply:
You decide if you want to update anything. I'm OK with this as-is.